enterprise_access/apps/api_client/tests/test_lms_client.py

```diff
@@ -0,0 +1,39 @@
+"""
```

Comment:
If we are going to be more microservice oriented, I think it would be nice to have an abstract api client that can be instantiated with a handful of values depending on the service we're going to talk to (LM, enterprise-catalog, LMS, ecom, and so on)

i think someone was talking about a slumber library or something that exists 🤔 -- i'll ask around, but yeah, i think longer term we're going to want something more general

Reply:
Could you add to the tech debt tracker please?

Reply:
added